Core/COEditingContext.m

```diff
@@ -947,6 +964,9 @@ cache in the same way on insertion and undeletion (insertion
 		__unused COPersistentRoot *loaded = [_loadedPersistentRoots objectForKey: persistentRootUUID];
 
 		// TODO: [self unloadPersistentRoot: loaded]
+		// Temporary hack until we have -unloadPersistentRoot:
+		[_lastTransactionIDForPersistentRootUUID removeObjectForKey: persistentRootUUID];
```

Comment:
I'd favour doing that in another PR, since this one is pretty complex already.
But feel free to add it to this PR if you feel like implementing it now.